### PR TITLE
CXSPA-3250: Add extensibility test for asm b2b customer list

### PR DESCRIPTION
**Pull request by i53577**
JIRA: https://jira.tools.sap/browse/CXSPA-3250

### CODE REVIEW

File: projects/storefrontapp-e2e-cypress/cypress/e2e/accessibility/asm-b2b-customer-list-tabbling-order.e2e.cy.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import { POWERTOOLS_BASESITE } from '../../sample-data/b2b-checkout';
+
+context('Tabbing order for ASM B2b Customer List', () => {
+  before(() => {
+    cy.window().then((win) => win.sessionStorage.clear());
+    Cypress.env('BASE_SITE', POWERTOOLS_BASESITE);
+  });
+
+  after(() => {
+    Cypress.env('BASE_SITE', ELECTRONICS_BASESITE);
```
Comment (niehuayang) [inline projects/storefrontapp-e2e-cypress/cypress/e2e/accessibility/asm-b2b-customer-list-tabbling-order.e2e.cy.ts:19]: So,  Can it also be deleted?

File: projects/storefrontapp-e2e-cypress/cypress/e2e/accessibility/asm-b2b-customer-list-tabbling-order.e2e.cy.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+import { tabbingOrderConfig as config } from '../../helpers/accessibility/tabbing-order.config';
+import { asmTabbingOrderWithCustomerList } from '../../helpers/accessibility/tabbing-order/asm';
+import { ELECTRONICS_BASESITE } from '../../helpers/checkout-flow';
+import { POWERTOOLS_BASESITE } from '../../sample-data/b2b-checkout';
+
+context('Tabbing order for ASM B2b Customer List', () => {
+  before(() => {
+    cy.window().then((win) => win.sessionStorage.clear());
+    Cypress.env('BASE_SITE', POWERTOOLS_BASESITE);
```
Comment (niehuayang) [inline projects/storefrontapp-e2e-cypress/cypress/e2e/accessibility/asm-b2b-customer-list-tabbling-order.e2e.cy.ts:15]: setting basesite maybe unnecessary, here is a suitable command,please see: https://github.com/SAP/spartacus/blob/8fa9315dcae04544a753719f1f88f05b4682ab52/projects/storefrontapp-e2e-cypress/package.json#L11

File: projects/storefrontapp-e2e-cypress/cypress/e2e/accessibility/asm-b2b-customer-list-tabbling-order.e2e.cy.ts [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  });
+
+  after(() => {
+    Cypress.env('BASE_SITE', ELECTRONICS_BASESITE);
+  });
+
+  describe('ASM', () => {
+    it('should allow to navigate with tab key for customer List', () => {
+      cy.cxConfig({
+        context: {
```
Comment (niehuayang) [inline projects/storefrontapp-e2e-cypress/cypress/e2e/accessibility/asm-b2b-customer-list-tabbling-order.e2e.cy.ts:25]: setting basesite maybe unnecessary, here is a suitable command,please see: https://github.com/SAP/spartacus/blob/8fa9315dcae04544a753719f1f88f05b4682ab52/projects/storefrontapp-e2e-cypress/package.json#L11

**Review by niehuayang — DISMISSED** [review]
approved and left a small suggestion.

File: projects/storefrontapp-e2e-cypress/cypress/e2e/accessibility/asm-b2b-customer-list-tabbling-order.e2e.cy.ts (hunk 3 above)
Comment (i53577) [reply projects/storefrontapp-e2e-cypress/cypress/e2e/accessibility/asm-b2b-customer-list-tabbling-order.e2e.cy.ts:25]: the setting is to tell pipeline which site should be used. I will contact you later this afternoon to explain it.

**Review by niehuayang — APPROVED** [review]

**Pull request merged**